api/src/main/java/io/strimzi/api/kafka/model/balancing/BrokerCapacity.java

```diff
@@ -72,6 +73,19 @@ public void setCpuUtilization(Integer cpuUtilization) {
         this.cpuUtilization = cpuUtilization;
     }
 
+    @Minimum(0)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Pattern("^\\d+m|\\d*(\\.\\d{0,3})*$")
+    @Description("Broker capacity for CPU resource in cores or milliCPU. " +
+            "For example, 1, 1.500, 1500m.")
```

Comment:
```suggestion
    @Description("Broker capacity for CPU resource in cores or millicores. " +
            "For example, 1, 1.500, 1500m.")
```

Should we link here to Kubernetes docs for the details about the units?